src/components/Header/social.js

```diff
@@ -0,0 +1,64 @@
+import * as SocialButtons from '../Buttons/Social';
+
+const SocialButtonsBuilder = (baseProps) => {
+  const iconBaseProps = baseProps;
+  const buttons = [];
+  const create = (config, name, optConfig) => {
+    if (config && config.length > 0) {
+      const btn = SocialButtons[name];
```

Comment:
Codacy found an issue: [Unable to validate computed reference to imported namespace 'SocialButtons'.](https://app.codacy.com/manual/matfilipowicz/BooGi/pullRequest?prid=5801182)